### PR TITLE
APIGOV-19118 update axway central CLI get command output to show updated r…

### PR DESCRIPTION
**Pull request by rsavajiyani**
…esource types

Thank you for your contribution to the Axway-Open-Docs repo.

## Describe the changes

Updated the “axway central get” command sample output to match the new output formats(with new columns for Resource Groups). 

Added a sample stage.json file users can use to create a new stage via the CLI.

## Deploy preview link

https://deploy-preview-1931--axway-open-docs.netlify.app/docs/central/cli_central/cli_create_fetch_resources/

https://deploy-preview-1931--axway-open-docs.netlify.app/docs/central/cli_central/cli_apiservices/

_Although Netlify will add this link below, it will redirect to the main page of the documentation, not to an specific page._

## Checklist for contributors

Before submitting this PR, please make sure:

* [X] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [X] You have signed the [Axway CLA](https://cla.axway.com/)
* [X] You have verified the technical accuracy of your change
* [X] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [X] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)
* [x] You have verified that all status checks have passed

_Put an x in the boxes that apply. This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@rsavajiyani I can review and merge this PR once @robertengaxway approves your changes.

File: content/en/docs/central/cli_central/cli_create_fetch_resources.md [hunk 1]
```diff
@@ -104,33 +104,61 @@ axway central get secrets
 Resource(s) successfully retrieved:
 
 ```
-NAME        AGE           TITLE       RESOURCE KIND  SCOPE KIND   SCOPE NAME
-secretname  4 months ago  secrettitle Secret         Environment  apigtw-v77
+NAME        AGE           TITLE       RESOURCE KIND  SCOPE KIND   SCOPE NAME    RESOURCE GROUP
+secretname  4 months ago  secrettitle Secret         Environment  apigtw-v77    management
+```
+
+### Stages
```
Comment (robertengaxway) [inline content/en/docs/central/cli_central/cli_create_fetch_resources.md:111]: Lets remove the Stages until we are have a better example to describe the use Stages associated with Environments in the next iteration of the docs.

File: static/samples/central/stage.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
+{
```
Comment (robertengaxway) [inline static/samples/central/stage.json:1]: Remove this example with stages

**Review by robertengaxway — APPROVED** [review]
approve as long as we remove the stage example until next iteration of the docs

File: static/samples/central/stage.json (hunk 2 above)
Comment (rsavajiyani) [reply static/samples/central/stage.json]: removed

File: content/en/docs/central/cli_central/cli_create_fetch_resources.md (hunk 1 above)
Comment (rsavajiyani) [reply content/en/docs/central/cli_central/cli_create_fetch_resources.md]: removed

**Comment by rsavajiyani** [comment]
> approve as long as we remove the stage example until next iteration of the docs

I removed the stage example file and the reference to it, thanks for your review/approval @robertengaxway

**Review by lbadenhop — APPROVED** [review]

**Pull request merged**